src/jobs/deploy.yml

```
@@ -21,7 +21,16 @@ steps:
         gcloud auth activate-service-account --key-file=gcloud-service-key.json
         gcloud config set project zen-formations
         gcloud config set app/use_appengine_api false
-        gcloud config set app/promote_by_default false
+  - unless:
+      conditions:
+        matches:
+          pattern: ^(main|master)$
+          value: << pipeline.git.branch >>
+      steps:
+        - run:
+            name: Do not promote unless main branch
+            command: |
+              gcloud config set app/promote_by_default false
```

I suggest simply:

```suggestion
        - run: gcloud config set app/promote_by_default false
```

Or if you really want to keep the name:

```suggestion
        - run:
            name: Do not promote unless main branch
            command: gcloud config set app/promote_by_default false
```

---

By habits when writing commands in YAML I use `|` to avoid any escaping issue.
But OK to remove it `|`.